lib/collection/src/grouping/types.rs

```diff
@@ -56,9 +55,6 @@ pub struct QueryGroupRequest {
 
     /// Limit of groups to return
     pub groups: usize,
-
-    /// Options for specifying how to use the group id to lookup points in another collection
-    pub with_lookup: Option<WithLookup>,
```

Comment:
we didn't need `with_lookup` here, it was not used and lookup was resolved before creating this structure